blocks/identity-block/package.json

```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "@arc-fusion/prop-types": "^0.1.5",
     "@arc-publishing/sdk-identity": "^1.79.0",
-    "react-google-recaptcha": "^3.1.0"
+    "react-google-recaptcha": "^3.1.0",
+    "react-google-recaptcha-v3": "^1.10.1"
```

Comment:
Make sure these are included as devDependencies in the root package.json, otherwise unit tests will fail (and they are in the test and lint action).